viper.go

```diff
@@ -1734,6 +1734,14 @@ func (v *Viper) getConfigFile() (string, error) {
 	return v.getConfigFile()
 }
 
+// Returns the path where Viper actually found the logfile it uses.
```

Comment:
A few things:

1. Godoc comments should begin with the element name.  `// GetConfigFilePath returns the path ...`.  See https://blog.golang.org/godoc-documenting-go-code.

1. You say *"where Viper actually found the **logfile** ..."*  Shouldn't that be *"configuration file"*?

2. I'd suggest removing the last two sentences describing use cases.  Just describe what the function does.

Reply:
I updated the PR. Hope everything is find, now